projects/qgrid-core/column-type/date.column.js

```diff
@@ -21,6 +24,14 @@ export class DateColumnModel extends DataColumnModel {
 		super('date');
 
 		this.format = 'MM/dd/yyyy';
+
+		this.label = function (row) {
+			const value = getValue(row, this);
+			const parse = parseFactory('date');
+			const date = parse(value);
+			const isValidDate = getType(date) === 'datetime' && !isNaN(date);
```

Comment:
why we need this?

Reply:
sometimes date parser could return an Invalid Date object (for not valid strings etc). In that case app crashes in date pipe

Reply:
please use 	qGridDate pipe instead of angular date pipe, there you don't need to check on isValidDate